inc/Widget/SmailyWidget.php

```diff
@@ -155,46 +155,252 @@ public function form( $instance ) {
+					<?php foreach ( $autoresponder_list as $autoresponder ) : ?>
+						<option value="<?php echo htmlentities( json_encode( $autoresponder ) ); ?>"<?php if ( $autoresponder['id'] == $current_autoresponder['id'] ) : ?> selected<?php endif; ?>><?php echo $autoresponder['name']; ?></option>
+					<?php endforeach; ?>
+					<?php endif; ?>
+				</select>
+				<p>
+					<?php echo esc_html_e( 'Select a suitable form-submitted trigger automation workflow from the list or create a new autoresponder on your Smaily account to be added to the list.', 'smaily' ); ?>
+				</p>
+			</div>
+			<?php endif; ?>
+
+			<!-- Layouts. -->
+			<div class="section">
+				<h2>
+					<?php echo esc_html_e( 'Layout', 'smaily' ); ?>
+				</h2>
+				<p>
+					<?php echo esc_html_e( 'Choose a preferred layout.', 'smaily' ); ?>
+				</p>
+
+				<div class="smaily-layout-container">
+					<p> <!-- Select layout. -->
+						<input 
+							class="radio" 
+							id="<?php echo $this->get_field_id( 'smaily-layout-1' ); ?>" 
+							name="<?php echo $this->get_field_name('form_layout'); ?>"
+							type="radio" value="smaily-layout-1" 
+							<?php if ( $form_layout === 'smaily-layout-1' ) { echo 'checked="checked"'; } ?> />
+						<label for="<?php echo $this->get_field_id( 'smaily-layout-1' ); ?>">
+						<strong><?php echo esc_html_e( 'Layout 1', 'smaily' ); ?></strong> &minus; <?php echo esc_html_e( 'email address with spaced button', 'smaily' ); ?></label>
+					</p>
+					<p>
+						<img src="<?php echo SMAILY_PLUGIN_URL . "static/layouts/email-layout1.svg"; ?>" alt="" title="<?php echo esc_attr_e( 'Layout 1', 'smaily' ); ?> &minus; <?php echo esc_attr_e( 'email address with spaced button', 'smaily' ); ?>" class="smaily-layouts">
+					</p>
+				</div>
+				<div class="smaily-layout-container">
+					<p>
+						<input 
+							class="radio" 
+							id="<?php echo $this->get_field_id( 'smaily-layout-2' ); ?>" 
+							name="<?php echo $this->get_field_name('form_layout'); ?>"
+							type="radio" value="smaily-layout-2" 
+							<?php if ( $form_layout === 'smaily-layout-2' ) { echo 'checked="checked"'; } ?> />
+						<label for="<?php echo $this->get_field_id( 'smaily-layout-2' ); ?>">
+						<strong><?php echo esc_html_e( 'Layout 2', 'smaily' ); ?></strong> &minus; <?php echo esc_html_e( 'email address with attached button', 'smaily' ); ?></label>
+					</p>
+					<p>
+						<img src="<?php echo SMAILY_PLUGIN_URL . "static/layouts/email-layout2.svg"; ?>" alt="" title="<?php echo esc_attr_e( 'Layout 2', 'smaily' ); ?> &minus; <?php echo esc_attr_e( 'email address with attached button', 'smaily' ); ?>" class="smaily-layouts">
+					</p>
+				</div>
+				<div class="smaily-layout-container">
+					<p>
+						<input 
+							class="radio" 
+							id="<?php echo $this->get_field_id( 'smaily-layout-3' ); ?>" 
+							name="<?php echo $this->get_field_name('form_layout'); ?>"
+							type="radio" 
+							value="smaily-layout-3" 
```

Comment:
Let's update the values of `form_layout`.

Having `smaily-` prefix in the value is redundant, because `form_layout` can only have form layouts. I think it would be better if the values just use `layout-[N]` pattern.